samples/error-correction/syndrome/Syndrome.qs

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+namespace Microsoft.Quantum.Samples.ErrorCorrection.Syndrome
+{
+    open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Intrinsic;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Measurement;
+    open Microsoft.Quantum.Arrays;
+    open Microsoft.Quantum.Diagnostics;
+    open Microsoft.Quantum.Convert;
+
+    /// # Summary
+    /// Measure qubit in a given basis and return the result
+    ///
+    /// # Input
+    /// ## basis
+    /// Pauli basis in which to perform measurement
+    /// ## qubit
+    /// Qubit to measure
+    /// 
+    /// # Output
+    /// ## result
+    /// Measurement result
+    operation BasisMeasure(basis: Pauli, qubit: Qubit): Result {
+        let result = Measure([basis], [qubit]);
+        return result;
+    }
+
+    /// # Summary
+    /// Prepare qubit in a given basis. Optionally flip the qubit if the value is True (One).
+    ///
+    /// # Input
+    /// ## qubit
+    /// Qubit to prepare
+    /// ## value
+    /// Value to prepare the qubit in (True for One, False for Zero)
+    /// ## basis
+    /// Basis to prepare the qubit in
+    operation PrepareInBasis(qubit: Qubit, value: Bool, basis: Pauli): Unit {
```

Comment:
The order of inputs here is different from the ordering used in `BasisMeasure`, such that I'd recommend using a common ordering for both. In particular, the [API principles](https://docs.microsoft.com/quantum/resources/contributing/api-design-principles) suggest ordering inputs based on commonality of partial application, which is almost always classical inputs before quantum. In this case, that'd point to `value : Bool, basis : Pauli, qubit : Qubit`, I think.

While the API principles don't apply to samples per se, looking at them can help make sure that users learning from your sample can transfer that knowledge to other parts of the Quantum Development Kit efficiently.